examples/cndpfwd/parse-args.c

```diff
@@ -196,12 +202,11 @@ process_callback(jcfg_info_t *j __cne_unused, void *_obj, void *arg, int idx)
             }
             pcfg.pi = umem->rinfo[lport->region_idx].pool;
 
-            if (lport->flags & LPORT_UNPRIVILEGED) {
-                if (f->xdp_uds)
-                    pcfg.xsk_uds = f->xdp_uds;
-                else
-                    CNE_ERR_RET("UDS info struct is null\n");
+            if (f->xdp_uds) {
+                pcfg.xsk_uds = f->xdp_uds;
+                lport->flags |= LPORT_UNPRIVILEGED;
```

Comment:
Maybe it's better if the lport flag is configured in the core library rather than here (more under the hood rather than having user responsibility to set it)...

Reply:
I will add a separate issue for this and assign it to me ;)